dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.internal.optional.jaxb-2.2.feature

```diff
@@ -0,0 +1,16 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.internal.optional.jaxb-2.2
+visibility=private
+IBM-App-ForceRestart: uninstall, \
+ install
+IBM-Process-Types: client, \
+ server
+Subsystem-Name: Java XML Bindings 2.2 for Java 9 and above
+-features=\
+  com.ibm.websphere.appserver.classloading-1.0
+-bundles=\
+  com.ibm.websphere.javaee.jaxb.2.2; required-osgi-ee:="(&(osgi.ee=JavaSE)(version>=9))"; location:="dev/api/spec/,lib/"; mavenCoordinates="javax.xml.bind:jaxb-api:2.2.12", \
```

Comment:
Shouldn't these technically use >=11?  My understanding is the jaxb stuff is going away in java 11.

Reply:
It is gone from the JDK completely in Java 11, but it is not on the default bootcp in Java 9/10.  I don't want to specify `--add-modules java.se.ee` for just 9/10 and have to do something different for 11+.  So this approach will give us consistent behavior accross 9/10/11